styled/resources.js

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components/native';
+import { Subhead, Title, Body } from '../components/BaseComponents';
+
+export const Card = styled.View`
+  border-bottom-width: 1px;
+  border-color: #ebebeb;
+  padding: 10px;
+  flex: 1;
+  flexdirection: column;
+  justifycontent: space-between;
+`;
+
+export const ContentContainer = styled.View`
+  flex-direction: column;
+  flex: 1;
+  justifycontent: flex-start;
+`;
+
+export const IconContainer = styled.View`
+  flex-direction: row;
+  flex: 4;
+  justifycontent: flex-end;
+  align-content: center;
+  align-items: center;
+`;
+
+export const LinkSubhead = styled(Subhead)``;
+
+export const ContentText = styled(Body)`
+  flex-direction: column;
+  flex: 1;
+  font-weight: normal;
+  color: black;
+`;
+
+export const CategoryCard = styled.View`
+  background-color: #ebebeb;
+  padding: 10px;
+  flex-direction: row;
+  flex: 1;
```

Comment:
Instead, try giving this a pixel height (60px from Figma ) and add `align-items: center` to vertically center this in the grey bar.